shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -95,6 +99,12 @@ public boolean supportReauthSignoutEnabled() {
                 .equals("true");
     }
 
+    public boolean supportAuthenticationAttemptsMigration() {
```

Comment:
Can we change the name for this to `isAuthenticationAttemptsServiceEnabled` as per the naming standard we discussed in dev chat a few weeks ago.